### PR TITLE
feat: add feature flags reset button

### PR DESCRIPTION
**Pull request by cjthompson**
## Description

feat: add feature flags reset button

## Notice

Before submitting a pull request, please make sure you have answered the following:

- [X] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [X] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

## Risk

None to users

## Testing

1. Press ALT+SHIFT+F
2. Change a flag
3. Click Reset App
4. Press ALT+SHIFT+F
5. Press Reset Feature Flags

## Screenshots (if applicable)
image

### CODE REVIEW

File: src/pages/Flags/Flags.tsx [hunk 1]
```diff
@@ -37,6 +38,14 @@ export const Flags = ({ route }: FlagsPageProps) => {
     }
   }
 
+  const handleResetPrefs = async () => {
+    try {
+      store.dispatch(slices.preferences.actions.clear())
```
Comment (0xdef1cafe) [inline src/pages/Flags/Flags.tsx:43]: you can do

```
const dispatch = useDispatch()
dispatch(....)
```

since we're inside a component

**Review by 0xdef1cafe — APPROVED** [review]

File: src/pages/Flags/Flags.tsx [hunk 2]
```diff
@@ -48,7 +57,10 @@ export const Flags = ({ route }: FlagsPageProps) => {
           </Stack>
         </Card.Body>
       </Card>
-      <Button onClick={handleReset}>Reset App</Button>
+      <HStack my={4}>
+        <Button onClick={handleReset}>Reset App</Button>
+        <Button onClick={handleResetPrefs}>Reset Flags to Default</Button>
```
Comment (0xdef1cafe) [inline src/pages/Flags/Flags.tsx:62]:
```suggestion
        <Button onClick={handleReset}>Apply</Button>
        <Button onClick={handleResetPrefs}>Reset Flags to Default</Button>
```

**Review by 0xdef1cafe — APPROVED** [review]
approved w/ suggestions cos this is internal only

File: src/pages/Flags/Flags.tsx (hunk 2 above)
Comment (cjthompson) [reply src/pages/Flags/Flags.tsx:62]: So the flag TECHNICALLY applies as soon as you change it, but for a user there's no need to understand the difference.

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**